src/Compilers/CSharp/Test/Semantic/Semantics/RefLocalsAndReturnsTests.cs

```diff
@@ -311,9 +311,9 @@ class C
                 // (13,36): error CS8333: Cannot return variable 'in int' by writable reference because it is a readonly variable
                 //     ref int M4(in int rro) => ref (rro = ref _rw);
                 Diagnostic(ErrorCode.ERR_RefReturnReadonlyNotField, "rro = ref _rw").WithArguments("variable", "in int").WithLocation(13, 36),
-                // (15,47): error CS0191: A readonly field cannot be assigned to (except in a constructor or a variable initializer)
+                // (15,47): error CS0192: A readonly field cannot be used as a ref or out value (except in a constructor)
```

Comment:
>// (15,47): error CS0192: A readonly field cannot be used as a ref or out value (except in a constructor) [](http://example.com/codeflow?start=16&length=105)

A change for this scenario is unexpected.

Reply:
Why? Previously it said `_ro` cannot be assigned to; but `_ro` isn't assigned to; it's used. So previously the error message was confusing, now it's correct.

Reply:
Perhaps.